dolphinscheduler-standalone-server/src/main/resources/common.properties

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
Is it necessary to add this file?

Reply:
HDFS is not enabled by default. In order to make the resource test run normally. The following two parameters need to be set in common.properties.
```
resource.storage.type=HDFS
fs.defaultFS=hdfs://hdfs:8020
``

Do you have a better solution?

Reply:
PTAL @kezhenxu94

Reply:
We don't modify default configs ONLY for tests, please move this file to test scope folder, and mount it into the docker container for tests.